packages/assistant-electron/copy-files.js

```diff
@@ -1,6 +1,6 @@
 /**
 * Software Name : UUV
-* SPDX-FileCopyrightText: Copyright (c) 2022-2023 Orange
+* SPDX-FileCopyrightText: Copyright (c) 2023-2024 Orange
```

Comment:
You can set 2022-2024 Here and other SPDX-FileCopyrightText 😉

Reply:
I've done it, you can check it.